src/braket/timings/time_series.py

```diff
@@ -87,6 +88,72 @@ def _ensure_sorted(self) -> None:
             self._series = OrderedDict(sorted(self._series.items()))
             self._sorted = True
 
+    @staticmethod
+    def constant_like(other_time_series: TimeSeries, constant: float = 0.0) -> TimeSeries:
```

Comment:
Maybe since the values from the other time series aren't used, I think it would be better to pass in a list of times instead. 

Since we're at it, maybe we should have a function that takes a list of times and values and a default value? That way you can use it in concatenate too? It seems like that would be more general. Do you think it's a good idea?

Reply:
Yeah, I modified the interface. Now creating TimeSeries from list of time points and a constant value, this feels more natural